COLLADAMaya/scripts/COLLADAMaya.vcxproj

```diff
@@ -1730,7 +1730,7 @@ copy "$(ProjectDir)*.mel" "%25MAYA_PATH2014_X64%25\scripts\others"
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='DebugPlugin2015|x64'">
     <PreBuildEvent>
-      <Command>call create_version_info_h.bat $(PLATFORM) $(CONFIGURATION)</Command>
+      <Command>call modify_version_info_h.bat</Command>
```

Comment:
I would not automatically call the script in the build script.
The end user can call the script as needed
Automatic build system can call the script - or their own version of this script - as needed. For automatic build system, calling this script in the build may in fact be a problem